charts/mtav7.x/templates/03-configmap_mta-analysis-v7-props.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+data:
+  application.properties: |
+    quarkus.kogito.devservices.enabled=false
+
+    mta.url = ${MTA_URL:http://tackle-ui.my-konveyor-operator.svc.cluster.local:8080}
+    quarkus.rest-client.mta_json.url = ${mta.url}
+    quarkus.rest-client.notifications.url=${BACKSTAGE_NOTIFICATIONS_URL:http://backstage-backstage.rhdh-operator/api/notifications/}
+    quarkus.openapi-generator.mta_json.auth.bearerAuth.bearer-token=${MTA_HUB_TOKEN}
```

Comment:
What is this and how to generate one?
There is no such props in https://github.com/parodos-dev/serverless-workflows/blob/main/mta-v7.x/application.properties

Reply:
These were generated using make command with gen-manifests option.

https://github.com/parodos-dev/serverless-workflows/blob/main/Makefile#L173